javascript/packages/cli/src/actions/setup.ts

```diff
@@ -209,6 +263,12 @@ const latestPolkadotReleaseURL = async (
       return Boolean(obj);
     });
 
+    if (!release) {
+      console.log(
+        `In repo '${repo}', there is no release for: '${name}'! Exiting...`,
```

Comment:
Do we need to `throw` here, since the next line 
`const { tag_name } = release;` will generate an error.

Reply:
Ah - good catch. I meant to throw error here - but I never did. Thank you

Reply:
Ah - good catch. I meant to throw error here - but I never did. Thank you